docs/en/observability/observability-ui.asciidoc

```diff
@@ -21,7 +21,7 @@ metrics discovery to APM, {kib} is the portal for accessing these and other capa
 
 Within {kib}, the *Observability Overview* page contains a wide variety of charts
 displaying analytics relating to the components that help you make your systems
-observable: logs, metrics, APM, and uptime data.
+observable: logs, metrics, APM, and Uptime data.
```

Comment:
Uptime was capitalized in the last few docs that I read, should it be capitalized throughout?